resume.html

```diff
@@ -0,0 +1,515 @@
+                <div class="col-4">
+                    <h5>System Developer</h5>
+                    <p>AL_Rowad for IT Solutions</p>
+                    <p>UAE - Abu Dhabi</p>
+                </div>
+                <div class="col-4">
+                    <p> Designed and developed a web-based application “e-Correspondence System” using mod-plsql Oracle and web technologies
+                    (html, css, JavaScript, and Ajax), used by more than 10,000 users in government organizations, such as the Abu Dhabi
+                    Police and the Ministry of Interior among others.</p>
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row narrow-row">
+                <div class="col gray-div">
+            
+                </div>
+            </div>
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-8 sub-title">
+                    Jan 2007 - Jun 2008
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+                    <h5>System Developer</h5>
+                    <p>Kantakji Center - KedySoft</p>
+                    <p>Aleppo, Syria</p>
+                </div>
+                <div class="col-4">
+                    <p> Created and developed solutions and interactive websites using C#.NET 2005, ASP.NET 2005 & MSSQL Server 2005.</p>
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row narrow-row">
+                <div class="col gray-div">
+            
+                </div>
+            </div>
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-8">
+                   <h3> Professional Skill Set</h3>
+                </div>
```

Comment:
This whole section should just be a list:
```
                <ul>
                    <li>Entrepreurial mindset</li>
                    <li>Critical thinking</li>
                    <li>Attention to detail</li>
                    <li>Teamwork & collaboration</li>
                </ul>
```
With added CSS classes to make it match the design.

Reply:
(you want these to be all stacked on mobile, and spread horizontally on desktop)